internal/func.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const fl = require('../');
-const equality = (x, y) => typeof x[fl.equals] === 'function' ? x[fl.equals](y) : x === y;
+const equality = (x, y) => x && typeof x[fl.equals] === 'function' ? x[fl.equals](y) : x === y;
```

Comment:
the new function doesn't return `x`; isn't it the same as:

`const equality = (x, y) => (x && typeof x[fl.equals] === 'function') ? x[fl.equals](y) : x === y;`

[Operator precedence](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Operator_Precedence)